build/lib/release.sh

```diff
@@ -26,6 +26,13 @@
 # This is where the final release artifacts are created locally
 readonly RELEASE_STAGE="${LOCAL_OUTPUT_ROOT}/release-stage"
 readonly RELEASE_DIR="${LOCAL_OUTPUT_ROOT}/release-tars"
```

Comment:
This dir now seems named inconsistently; following the other two it should be RELEASE_TARS